pkg/granted/registry/add.go

```diff
@@ -33,8 +34,10 @@ var AddCommand = cli.Command{
 		&cli.BoolFlag{Name: "prefix-all-profiles", Aliases: []string{"pap"}, Usage: "Provide this flag if you want to append registry name to all profiles"},
 		&cli.BoolFlag{Name: "prefix-duplicate-profiles", Aliases: []string{"pdp"}, Usage: "Provide this flag if you want to append registry name to duplicate profiles"},
 		&cli.BoolFlag{Name: "write-on-sync-failure", Aliases: []string{"wosf"}, Usage: "Always overwrite AWS config, even if sync fails (DEPRECATED)"},
-		&cli.StringSliceFlag{Name: "required-key", Aliases: []string{"r", "requiredKey"}, Usage: "Used to bypass the prompt or override user specific values"}},
-	ArgsUsage: "--name <registry_name> --url <repository_url>",
+		&cli.StringSliceFlag{Name: "required-key", Aliases: []string{"r", "requiredKey"}, Usage: "Used to bypass the prompt or override user specific values"},
+		&cli.StringFlag{Name: "type", Aliases: []string{"t"}, DefaultText: "git", Usage: "specify the type of granted registry source you want to set up. Default: git"}},
```

Comment:
I think `--type` is fine (I'm hesitant to add aliases too aggressively because `-t` becomes reserved forever for it, I prefer for us to introduce them *only* for very frequently used commands initially)

```suggestion
		&cli.StringFlag{Name: "type", DefaultText: "git", Usage: "specify the type of granted registry source you want to set up. Default: git"}},
```